FutureNorthants/northants-website
# Updated packages, alter hero image config

### Setup
Checkout this branch and run a drush config:import

### Testing
1. Create a new service landing page. Verify there is now a Hero Image tab, but don't set one (and leave the empty fields in place, don't use the Remove button). Save in published state. Verify it allows the save without providing an image.
2. View your page in the frontend and verify it displays normally.
3. Edit the page in the CMS and add a hero image and save in published state.
4. View the page in the frontend and verify it now displays a hero image.
5. Edit a service page and verify the icon field is now the first one, as it is on service landing pages.
